ably.d.ts

```diff
@@ -853,19 +853,19 @@ declare namespace ChannelModes {
   /**
```

Comment:
Maybe worth leave a link to this pr or https://github.com/ably/ably-js/issues/1954 in the docstring comment for ChannelModes to explain the redundant values